modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -1390,6 +1390,22 @@ definition {
 		Click.clickAtNotVisible(locator1 = "ObjectAdmin#ENTRY_BOOLEAN_CHECKBOX");
 	}
 
+	macro moveDefinitionFolder {
+		Refresh();
```

Comment:
I think we can move the refresh to the test case level and not have it in this macro so we can reuse this even if the test will not need a refresh and it will only contain the steps specific to moving a definition to folder. Let me know what you think. Thanks!